test/e2e/storage/csi_mock_volume.go

```diff
@@ -1220,7 +1220,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 		}
 	})
 
-	ginkgo.Context("CSI Volume Snapshots [Feature:VolumeSnapshotDataSource]", func() {
+	ginkgo.Context("CSI Volume Snapshots", func() {
```

Comment:
We have to keep this feature gate.  Not all k8s clusters support volume snapshot.

Reply:
reverted